deployments/docker-production/docker-compose.yml

```diff
@@ -21,6 +21,7 @@ services:
         awslogs-group: lbrytv
         awslogs-region: us-east-1
         awslogs-stream: app
+    restart: unless-stopped
```

Comment:
do we have a mechanism to capture this? It will no longer be as obvious but it will be good to address these situations when it fails.

Reply:
Only by visual log inspection currently. This should be fixed as a part of #124